systemtest/scripts/deploy_prometheus_and_grafana.sh

```diff
@@ -0,0 +1,145 @@
+#!/bin/bash
+DIR=`pwd`
+k=false
+
+usage() { echo "Usage: $0 [-k] -n <namespace> -c <cluster>"  
+		  echo " When -k is used, Kafka cluster with metrics is deployed."
+		  1>&2; exit 1;}
+
+while getopts "kn:c:" o; do
+    case "${o}" in
+        k)
+            k=true
+            ;;
+        n)  NAMESPACE=${OPTARG}
+        	;;
+        c)  CLUSTER=${OPTARG}
+        	;; 	
+        *)
+            usage
+            ;;
+    esac
+done
+shift $((OPTIND-1))
+
+if [ -z "${NAMESPACE}" ] || [[ "$k" == true && -z "${CLUSTER}" ]] ; then
+    usage
+fi
+
+if [ "$k" == true ] ; then
+	# Determine whether CO is deployed in specified NS
+	lines_of_operator=`oc get pods -l strimzi.io/kind=cluster-operator -n $NAMESPACE | wc -l`
+
+	if [ "$lines_of_operator" -lt "2" ]; then
+		echo "Cluster Operator is not deployed!"
+		exit 1
+	fi
+
+	echo "Deploying Kafka "$CLUSTER
+	sed "s/my-cluster/$CLUSTER/" $DIR/metrics/examples/kafka/kafka-metrics.yaml > $DIR/metrics/examples/kafka/$CLUSTER-kafka-metrics.yaml
+
+	oc apply -f $DIR/metrics/examples/kafka/$CLUSTER-kafka-metrics.yaml -n $NAMESPACE
+
+	# Delay for allowing cluster operator to create the Zookeeper statefulset
+	sleep 5
+
+	zkReplicas=$(oc get kafka $CLUSTER -o jsonpath="{.spec.zookeeper.replicas}" -n $NAMESPACE)
+	echo "Waiting for Zookeeper cluster to be ready..."
+	readyReplicas="0"
+	while [ "$readyReplicas" != "$zkReplicas" ]
+	do
+	    sleep 2
+	    readyReplicas=$(oc get statefulsets $CLUSTER-zookeeper -o jsonpath="{.status.readyReplicas}" -n $NAMESPACE)
+	done
+	echo "...Zookeeper cluster ready"
+
+	# Delay for allowing cluster operator to create the Kafka statefulset
+	sleep 5
+
+	kReplicas=$(oc get kafka $CLUSTER -o jsonpath="{.spec.kafka.replicas}" -n $NAMESPACE)
+	echo "Waiting for Kafka cluster to be ready..."
+	readyReplicas="0"
+	while [ "$readyReplicas" != "$kReplicas" ]
+	do
+	    sleep 2
+	    readyReplicas=$(oc get statefulsets $CLUSTER-kafka -o jsonpath="{.status.readyReplicas}" -n $NAMESPACE)
+	done
+	echo "...Kafka cluster ready"
+
+	echo "Waiting for entity operator to be ready..."
+	oc rollout status deployment/$CLUSTER-entity-operator -w -n $NAMESPACE
+	echo "...entity operator ready"
+
+	rm $DIR/metrics/examples/kafka/$CLUSTER-kafka-metrics.yaml
+
+fi
+
+# Not deploying Kafka
+# Deploy Prometheus operator
+curl https://gist.githubusercontent.com/stanlyDoge/5a0b43114d9be0e72f0f4b4e3ee168a1/raw/09614357fa20efd14deffb31a06fd87aaa6dc6c8/prometheus-operator.yaml --output $DIR/metrics/prometheus-operator.yaml
+
+sed "s/namespace: .*/namespace: $NAMESPACE/" $DIR/metrics/prometheus-operator.yaml > $DIR/metrics/prometheus-operator-deploy.yaml
```

Comment:
You should use the multiplatform support to make this work also on MacOS (not just here but accross the script). Or does htis run inside some container and not on local environment where the tests are started?

Reply:
No. I understand the QE folks want a simple script to deploy stuff for them. I guess it is for all the manual testing.

Reply:
Right. So then it either needs proper multiplatform support to be merged and maybe it should be int he tools directory. Or they should just keep it in their _hacking_ directories and use it from there. But we need to understand why is this needed before we merge this.

Reply:
@Frawless ^

Reply:
It's for testing grafana dashboards and things around prometheus and as Standa mentioned, it's mostly for manual checks, cause check, that grafana dashboards are written properly has to be done manually atm. Basically, we just wanted to save some time. If it's problem to have it here, we can store it on other place.

Reply:
Well, we should then either do it properly and add it to place where everyone can find it and use it or we should keep it somewhere else.

Reply:
so what do you suggest? Move it from systemtests scripts and put it to some `hacking` directory which will be mentioned in `HACKING.md` ?

Reply:
We have the tools directory. But first it needs to address the comments anyway.